package.json

```diff
@@ -29,14 +30,22 @@
     "ts-node": "^10.9.1",
     "typescript": "^4.1.3"
   },
+  "lint-staged": {
+    "*.{js,ts,json}": [
+      "yarn lint:fix",
+      "git add"
```

Comment:
From my understanding of this package, we don't need this command, as it's the job of lint-staged